lib/features/recipes/presentation/recipe_screen.dart

```diff
@@ -1,15 +1,93 @@
 import 'package:flutter/material.dart';
 
+import '../../../common/widgets/base_scaffold.dart';
+import '../../../db/recipe.dart';
+import '../../home/presentation/widgets/difficulty_indicator.dart';
+
 class RecipeScreen extends StatefulWidget {
-  const RecipeScreen({super.key});
+  final int id;
```

Comment:
Die Id wird ja oben in der AppBar angezeigt, aber wenn man swiped wird die nicht geupdatet oder? Also auch nicht wenn die Rezepte aus der DB gezogen werden. Brauchen wir das überhaupt wenn darunter der Tag des Rezeptes angezeigt wird?